vent/coordinator/coordinator.py

```diff
@@ -200,7 +160,14 @@ def stop(self):
         Stop the coordinator.
         This does a soft stop (not kill a process)
         """
-        self.rpc_client.stop()
+        try:
+            self.rpc_client.stop()
+        except ConnectionRefusedError:
```

Comment:
Can this exception possible raise from rpc_client?

Reply:
yes this was the exception I was getting here.